scripts/sg-ticket-link.coffee

```diff
@@ -28,5 +28,9 @@ module.exports = (robot) ->
     if isNaN(ticket_number)
       return
     
-    shotgun_url = process.env.HUBOT_SHOTGUN_URL
-    msg.send "#{shotgun_url}/detail/Ticket/#{ticket_number}"
+    if msg.message.match(/ZD#(\d+)/)
+      zendesk_url = process.env.HUBOT_ZENDESK_TICKET_URL
+      msg.send "#{zendesk_url}#{ticket_number}"
```

Comment:
Can't you just make sg-ticket-link shut up when the match is `ZD#`, and have the Zendesk script handle it? That way, you don't muddy up concerns.

Or better: change the regexp for this to force leading white-space: `/\s+#(\d+)/`